### PR TITLE
Finish first two cards - #1

### PR DESCRIPTION
**Pull request by mertdusunceli**
Closes the Trello card
https://trello.com/c/4x3DVAIn/312-issue-2-save-image-link

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -10,6 +10,28 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 0) do
+ActiveRecord::Schema.define(version: 2020_02_11_224457) do
+
+  create_table "articles", force: :cascade do |t|
```
Comment (ndelage) [inline db/schema.rb]: I think these two tables (articles and comments) were from the tutorial you worked on no? If they aren't part of the Ropes work, please remove them.

File: app/controllers/images_controller.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  end
+
+  def create
+    @image = Image.new(image_params)
+    if @image.save
+      redirect_to @image
+    else
+      render 'new'
+    end
+    # render plain: params[:article].inspect
```
Comment (ndelage) [inline app/controllers/images_controller.rb]: please delete this line. better to not commit any commented out code

File: app/controllers/images_controller.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  end
+
+  def show
+    @image = Image.find(params[:id])
+  end
+
+  def index
```
Comment (ndelage) [inline app/controllers/images_controller.rb]: This index route should be removed as well, it belongs in the next PR

File: app/views/images/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<h3>Welcome to imgur</h3>
```
Comment (ndelage) [inline app/views/images/index.html.erb]: ditto for this template, let's save this for the next PR (I can help you pull the `index` code out and save it for later)

File: db/schema.rb (hunk 1 above)
Comment (mertdusunceli) [reply db/schema.rb]: Done!

File: app/views/images/index.html.erb (hunk 4 above)
Comment (mertdusunceli) [reply app/views/images/index.html.erb]: Done

File: app/controllers/images_controller.rb (hunk 2 above)
Comment (mertdusunceli) [reply app/controllers/images_controller.rb]: Done

**Pull request merged**